lib/trento_web/controllers/v1/cluster_controller.ex

```diff
@@ -1,20 +1,21 @@
 defmodule TrentoWeb.V1.ClusterController do
   use TrentoWeb, :controller
+  use OpenApiSpex.ControllerSpecs
 
   alias Trento.Clusters
 
   alias TrentoWeb.OpenApi.Schema
 
-  use OpenApiSpex.ControllerSpecs
+  plug(OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true)
```

Comment:
Is `mix format` adding parenthesis here and the next line?
It looked nicer without XD

Reply:
argh, this is a formatter quirk that happens on certain conditions, it adds parenthesis even if you have `local_without_parens` enabled for that macro